src/pages/starter-kit/checkout/tax-reference.md

```diff
@@ -258,6 +258,14 @@ There are two GraphQL queries to check taxes applied by the tax integration, `ca
 
 To check the taxes applied to the cart, you can use the [`getCart`](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/queries/cart/) query to retrieve the [`cart`](https://developer.adobe.com/commerce/webapi/graphql-api/index.html#definition-Cart)/[`prices`](https://developer.adobe.com/commerce/webapi/graphql-api/index.html#definition-CartPrices)/[`applied_taxes`](https://developer.adobe.com/commerce/webapi/graphql-api/index.html#definition-CartPrices) field. This field contains information about the taxes applied to the cart.
 
+>[!TIP]
+>
+>To enable full tax breakdown, set `Stores > SETTINGS > Configuration > SALES > Tax > Shopping Cart Display Settings > Display Full Tax Summary` to `Yes`
+
 ### Order taxes
 
 To check the tax breakdown, you can use the [`customerOrders`](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/queries/orders/) query to retrieve the [`CustomerOrder`](https://developer.adobe.com/commerce/webapi/graphql-api/index.html#definition-CustomerOrder)/[`total`](https://developer.adobe.com/commerce/webapi/graphql-api/index.html#definition-OrderTotal)/[`taxes`](https://developer.adobe.com/commerce/webapi/graphql-api/index.html#definition-TaxItem) field. This field contains information about the taxes breakdown applied to the order.
+
+>[!TIP]
+>
+>To enable full tax breakdown, set `Stores > SETTINGS > Configuration > SALES > Tax > Orders, Invoices, Credit Memos Display Settings > Display Full Tax Summary` to `Yes`
```

Comment:
```suggestion
>To enable full tax breakdown, set **Stores** > Settings > **Configuration** > Sales > **Tax** > **Orders**, **Invoices** > **Orders, Invoices, Credit Memos Display Settings** > **Display Full Tax Summary**` to `Yes`.
```

Change tip above to match.

Reply:
It seems there's a backtick off, might be like this
```suggestion
>To enable full tax breakdown, set **Stores** > Settings > **Configuration** > Sales > **Tax** > **Orders**, **Invoices** > **Orders, Invoices, Credit Memos Display Settings** > **Display Full Tax Summary** to `Yes`.
```